<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Message</string>
     </attribute>
     <widget class="QGroupBox" name="gb_text">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Text</string>
      </property>
      <widget class="QTextEdit" name="text">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>551</width>
         <height>291</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="db_sending_settings">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>151</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Sending settings</string>
      </property>
      <widget class="QFrame" name="user_display">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>131</width>
         <height>61</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QRadioButton" name="ds_app">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Resource usage per unit limitaton</string>
        </property>
        <property name="text">
         <string>App</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="ds_null">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Absolute resource usage</string>
        </property>
        <property name="text">
         <string>Null</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QFrame" name="user_display_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>80</y>
          <width>91</width>
          <height>71</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QRadioButton" name="ds_abs_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Absolute resource usage</string>
         </property>
         <property name="text">
          <string>Database</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="ds_rel_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Resource usage per unit limitaton</string>
         </property>
         <property name="text">
          <string>App</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="ds_abs_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Absolute resource usage</string>
         </property>
         <property name="text">
          <string>Null</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QLabel" name="label_ud">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Usage:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_ud_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="send">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>131</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id_line">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="send_file">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>131</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Send file</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Validation</string>
     </attribute>
     <widget class="QGroupBox" name="gb_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Validation table</string>
      </property>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>551</width>
         <height>291</height>
        </rect>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>151</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Validation settins</string>
      </property>
      <widget class="QPushButton" name="view">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>131</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>View</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cb_abs_acc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Absolute accuracy</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cb_any_acc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Any accuracy</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cb_prc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Precision</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cb_rec">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Recall</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cb_f_score">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>F-score</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_type_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Table:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="db_val_combobox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>prc_articles_val</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="calculate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>131</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
